apps/mail/components/ui/navigation-menu.tsx

```
@@ -1,160 +1,275 @@
+										<a
+											className="flex h-full w-full select-none flex-col justify-end rounded-lg p-[25px] no-underline outline-none"
+											style={{
+												backgroundImage: "url('/couple.jpeg')",
+												backgroundSize: "cover",
+												backgroundPosition: "center",
+												backgroundRepeat: "no-repeat"
+											}}
+											href="/"
+										>
+											
+										</a>
```

Biome (2.1.2)

[error] Provide screen reader accessible content when using `a` elements.

All links on a page should have content that is accessible to screen readers.
Accessible content refers to digital content that is designed and structured in a way that makes it easy for people with disabilities to access, understand, and interact with using assistive technologies.
Follow these links for more information,
 WCAG 2.4.4
 WCAG 4.1.2

(lint/a11y/useAnchorContent)